tests/MailsMockTraitTest.php

```diff
@@ -118,6 +118,19 @@ public function testMailWithExport()
         $this->assertFileExists($this->getFixturePath('test_mail_with_export.html'));
     }
 
+    public function testMailWithGlobalExportMode()
+    {
+        putenv('FAIL_EXPORT_JSON=false');
+
+        Mail::to('test@mail.com')->queue(new TestMail(['name' => 'John Smith']));
+
+        $this->assertMailEquals(TestMail::class, [
+            $this->mockedMail('test@mail.com', 'test_mail_with_global_export.html', 'Test Subject'),
+        ]);
+
+        $this->assertFileExists($this->getFixturePath('test_mail_with_export.html'));
```

Comment:
why you checking `test_mail_with_export.html` file, but asserting content with the `test_mail_with_global_export.html`?